ledger/src/blockstore.rs

```diff
@@ -1010,6 +1021,10 @@ impl Blockstore {
             write_batch.put::<cf::ErasureMeta>(erasure_set.store_key(), &erasure_meta)?;
         }
 
+        for (erasure_set, merkle_root_meta) in merkle_root_metas {
+            write_batch.put::<cf::MerkleRootMeta>(erasure_set.key(), &merkle_root_meta)?;
+        }
```

Comment:
doesn't this overwrite existing merkle-root meta in blockstore?
it seems like at least some entries will be re-written with the same value.

Reply:
you're right, i just c/p the `erasure_metas` logic, but I think we can be smarter and keep track of which entries are dirty and need to be written.